fluent-dom/src/overlay.js

```diff
@@ -43,152 +43,148 @@ const LOCALIZABLE_ATTRIBUTES = {
 
 
 /**
- * Overlay translation onto a DOM element.
+ * Translate an element.
  *
- * @param   {Element} targetElement
- * @param   {string|Object} translation
+ * Translate the element's text content and attributes. Some HTML markup is
+ * allowed in the translation. The element's children with the data-l10n-name
+ * attribute will be treated as arguments to the translation. If the
+ * translation defines the same children, their attributes and text contents
+ * will be used for translating the matching source child.
+ *
+ * @param   {Element} element
+ * @param   {Object} translation
  * @private
  */
-export default function overlayElement(targetElement, translation) {
+export default function translateElement(element, translation) {
```

Comment:
The diff of `overlay.js` isn't very helpful, I'm afraid. I refactored most of the file. I may suggest opening the new version in a new tab and reading it as it is.